flows/definition/legacy.go

```diff
@@ -296,34 +308,33 @@ func createAction(baseLanguage utils.Language, a legacyAction, fieldMap map[stri
 			fieldMap[a.Value] = fieldUUID
 		}
 
-		translated, _ := excellent.TranslateTemplate(a.Value)
+		migratedValue, _ := excellent.MigrateTemplate(a.Value)
+
+		// flows now have different action for name changing
+		if a.Field == "name" || a.Field == "first_name" {
+			// we can emulate setting only the first name with an expression
+			if a.Field == "first_name" {
+				migratedValue = fmt.Sprintf("%s @(word_slice(contact.name, 2, -1))", migratedValue)
```

Comment:
We have a way of changing it in the old world? I thought we just set name as a whole?